### ISSUE TITLE
Machine Learning in Python using pandas, Numpy, Keras and various libraries

### ISSUE THREAD

**Issue by yugkhanna** [pydelhi/talks#94]
It will include a live presentation of data science examples from basics to advanced ( Numpy, Keras, Matplotlib, Seaborn, Pandas etc. ) and also machine learning concepts like Regression, Cost and Loss Functions and K-Maps ( won't give the spoilers right now)
My own Kaggle Machine Learning code ( Ranked 1100/12000 approx. ) in which I trained a data set and got upto 0.81 accuracy. I will explain the code in detail and take detailed questions.

It will take approx. 20-30 minutes.

Slides not made yet. You can check my GitHub for the code

I'm an Indian student studying Computer Engineering at University of Waterloo Canada and am here in India for the summer working for a company and serving as a software developer intern.

I think my experience with Python and Data Science and my skills will be helpful to people who want to go ahead in data science and people will be able to learn new skills and understand concepts better.

I already posted an issue before this but this is just an elaborated post as everyone I saw elaborated their proposals.
Hope you guys are interested in my talk. I will be in India from 17th May to August End.

**Comment by vsnehil92** [comment]
Hi @yugkhanna  , would you be available to give a talk on this topic for a 30/45 minutes session on [14th April meetup](https://www.meetup.com/pydelhi/events/lvkpknyxgbsb/)?

**Comment by yugkhanna** [comment]
Hi @vsnehil92 , I'm coming to India on 17th May. Any meetup after that? I have amazing stuff to share.

**Comment by yugkhanna** [comment]
I can do 26th May Meetup if that's okay?

**Comment by vsnehil92** [comment]
Are you available on 12th May?

**Comment by yugkhanna** [comment]
I mentioned in my proposal I'm coming to India on 17th May buddy. How can I deliver before that? I can do 26th May

**Comment by rajataaron** [comment]
Hi @yugkhanna, sure we will arrange your talk according to the given date schedule by you :)

**Comment by vsnehil92** [comment]
Ok, will let you know about 26th May.

**Comment by yugkhanna** [comment]
You'll have a great time with my presentation. Hyped to meet you guys

**Comment by yugkhanna** [comment]
Should I consider this as a yes and prepare for my talk on 26th May?

**Comment by rajataaron** [comment]
@yugkhanna yes you can start preparing for that and we will put you on 26th may schedule in the 1st or 2nd week of May.

**Comment by yugkhanna** [comment]
Perfect

**Comment by yugkhanna** [comment]
How will you guys connect my laptop to the projector? I have a macbook pro which only has USB-C ports. Do I need to bring a converter to HDMI or something?

Also prereqs for the presentation : Keras, Numpy, Pandas. Its better if they have it installed otherwise I will guide them how to.

**Comment by rajataaron** [comment]
If you have a converter HDMI to USB-C then it's awesome. Otherwise, we have to arrange it.

**Comment by rajataaron** [comment]
Can you make a small documentation containing an installation of these prereqs? So we will distribute than instructions to all so, on meetup day we can start it without any installation hustle.

**Comment by yugkhanna** [comment]
Sure

**Comment by yugkhanna** [comment]
I am attaching the documentation.

https://github.com/yugkhanna/Talk-PyDelhi/blob/master/Documentation.md

**Comment by yugkhanna** [comment]
Is the documentation okay?? Can you guys arrange a usb-c to hdmi converter? I am unable to arrange it right now!?

**Comment by rajataaron** [comment]
Yes I think documentation is good to go. 
And We will try to arrange usb-c to hdmi converter and will let you know before scheduling your talk.

**Comment by yugkhanna** [comment]
Thanks

**Comment by rajataaron** [comment]
Hi, @yugkhanna are you available for 9th Jun to deliver your talk? As we have a new format in PyDelhi one meetup is dedicated to Talks and other is to DevSprint and on 26th May we have dev sprint session.

**Comment by yugkhanna** [comment]
No issues!!

**Comment by rajataaron** [comment]
@yugkhanna we are looking for a venue once we are done with it we will arrange USB-C to HDMI converter and will update you accordingly.
Can we have your email id?

**Comment by yugkhanna** [comment]
yugkhanna1@gmail.com

**Comment by utkarsh2102** [comment]
Hi @yugkhanna.
We have your talk planned on this Saturday, 9th June.
And the venue has been updated for the same. Details of the venue are as follows:
`Fueled Noida`
`A-20, Block A, Sector-3, Noida`
The meetup will start at 11AM. Requesting you to be on time :)
Also, please reply to this message once acknowledged.

**Comment by yugkhanna** [comment]
Hey there!
Thanks for the update! As I have a MacBook Pro with only USB C , I request
you guys to arrange an adapter ( Usb c to hdmi ) if possible.

**Comment by rajataaron** [comment]
Hi @yugkhanna we have USB-C to HDMI adapter but the projector we have uses VGA cable.
So I suggest you take backup of your presentation on the web and make notes of PDF which you show to the attendees. And the second option we have by screen sharing on other systems via Teamviewer which is connected to the projector.

**Comment by vipulgupta2048** [comment]
@yugkhanna  Can you add a link to your slides that you made.

**Comment by rajataaron** [comment]
@CuriousLearner you can close this issue now. The talk has been delivered successfully.
